--- origsrc/QScintilla-1.71-gpl-1.7.1/designer/designer.pro	2006-11-30 03:32:15.000000000 -0600
+++ src/QScintilla-1.71-gpl-1.7.1/designer/designer.pro	2006-12-25 03:07:15.215375000 -0600
@@ -3,10 +3,11 @@
 
 TEMPLATE = lib
 TARGET = qscintillaplugin
-DESTDIR = $(QTDIR)/plugins/designer
+DESTDIR = ../plugins/designer
 
-CONFIG += qt warn_on release plugin
+CONFIG += qt warn_on release plugin thread
 
 SOURCES += qscintillaplugin.cpp
 
-LIBS += -lqscintilla
+INCLUDEPATH = ../qt
+LIBS += -L../lib -lqscintilla
--- origsrc/QScintilla-1.71-gpl-1.7.1/example/application.pro	2006-11-30 03:32:15.000000000 -0600
+++ src/QScintilla-1.71-gpl-1.7.1/example/application.pro	2013-07-24 04:12:55.239917300 -0500
@@ -3,7 +3,8 @@ TARGET		= application
 
 CONFIG		+= qt warn_on release
 
-LIBS		+= -lqscintilla
+INCLUDEPATH	+= ../qt
+LIBS		+= -L../lib -lqscintilla
 
 HEADERS		= application.h
 SOURCES		= application.cpp \
--- origsrc/QScintilla-1.71-gpl-1.7.1/qt/qscintilla.pro	2006-11-30 03:32:30.000000000 -0600
+++ src/QScintilla-1.71-gpl-1.7.1/qt/qscintilla.pro	2006-12-25 02:47:27.637250000 -0600
@@ -25,7 +25,7 @@
 
 TEMPLATE = lib
 TARGET = qscintilla
-DESTDIR = $(QTDIR)/lib
+DESTDIR = ../lib
 CONFIG += qt warn_off release dll thread
 INCLUDEPATH = . ../include ../src
 DEFINES = QEXTSCINTILLA_MAKE_DLL QT SCI_LEXER
